docs/spectrum1d.rst

```diff
@@ -87,6 +87,18 @@ A full list of the supported formats is shown in the table below.
 
 | More information on creating custom loaders can be found in the :doc:`custom loading </custom_loading>` page.
 
+Writing to a File
+-----------------
+
+Similarly, a `~specutils.Spectrum1D` object can be saved to any file format supported by
+:class:`~astropy.nddata.CCDData` via the :meth:`specutils.Spectrum1D.write` method.
+
+.. code-block:: python
+
+    >>> from specutils import Spectrum1D
+    >>> spec1d = Spectrum1D.read("/path/to/input.fits")  # doctest: +SKIP
```

Comment:
Do we really need this? Reading is already covered in https://specutils.readthedocs.io/en/stable/spectrum1d.html#reading-from-a-file

```suggestion
```

Reply:
Should the snippets be somewhat standalone or is it ok to assume at this point that the user already has a `spectrum1D` object loaded/created?

Reply:
Depends on who you ask. 😆 

You can `ref` back to the "reading" section if you want, but I think it is overkill to tell people how to read again.